.editorconfig

```diff
@@ -1,71 +1,155 @@
-# EditorConfig is awesome:http://EditorConfig.org
+# EditorConfig is awesome: http://EditorConfig.org
 
 # top-most EditorConfig file
 root = true
 
-# Don't use tabs for indentation.
 [*]
 indent_style = space
 # (Please don't specify an indent_size here; that has too many unintended consequences.)
+charset = utf-8
+trim_trailing_whitespace = true
+end_of_line = lf
+insert_final_newline = true
 
-# Code files
-[*.{cs,csx,vb,vbx}]
-indent_size = 4
-
-# Xml project files
-[*.{csproj,vbproj,vcxproj,vcxproj.filters,proj,projitems,shproj}]
+[*.{build,config,csproj,js,json,proj,props,targets,xml,ruleset,xsd}]
 indent_size = 2
 
-# Xml config files
-[*.{props,targets,ruleset,config,nuspec,resx,vsixmanifest,vsct}]
-indent_size = 2
+[*.{cs,cshtml,ascx,aspx}]
+indent_size = 4
 
-# JSON files
-[*.json]
-indent_size = 2
+#### C#/.NET Coding Conventions ####
 
-# Dotnet code style settings:
-[*.cs]
-# Sort using and Import directives with System.* appearing first
-dotnet_sort_system_directives_first = false
-insert_final_newline = true
+# 'using' directive preferences
+dotnet_sort_system_directives_first = true
```

Comment:
Placing System directives first is [common](https://github.com/dotnet/roslyn/blob/main/src/EditorFeatures/Core/CodeActions/CodeActionEditHandlerService.cs) in C# code. It is also the default for StyleCop, Resharper and Visual Studio, iirc. Because automated code cleanup rearranges them based on these settings, I thought it wouldn't be a big deal to change this. However I don't feel strongly about it and am okay with reverting the change.